src/limit-add.test.ts

```diff
@@ -0,0 +1,25 @@
+import { TestFixture, Expect, TestCase } from "alsatian";
+import { limitAdd } from "./limit-add";
+
+
+@TestFixture()
+export class LimitTest {
```

Comment:
This should be `LimitAddTests` - make sure it matches the name of the thing you're testing